### DISCUSSION TITLE
Option to close notification from code

### DISCUSSION THREAD

**Discussion by ahweis** [mantinedev/mantine#1305] [Q&A]
Currently the notification first closes on auto close or when the user clicks the close button.
It would be nice with the option to close a specific notification, e.g. if the notification contains a link to another page or an extended elaboration on an issue and the user get a modal presented with a message, then the notification stays until auto close or the user closes it.

**Comment by rtivital** [comment]
https://mantine.dev/others/notifications/#remove-notifications-from-state-and-queue

**Reply by ahweis** [reply]
Thanks - did not look in the notification system

**Comment by scott-wojan** [comment]
I'm looking for something similar and tried the hideNotification() option but it doesn't work for what I'm trying to do.  I just have a notification in a page like below and just want it to close when the default close button is clicked.
```
 <Notification
    title="Default notification">
    This is default notification with title and body
</Notification>
```

**Reply by Jufrench** [reply]
I was also looking for the same thing and couldn't find it. I ended up just using `useState` to determine whether or not it should be displayed. Then you could set the state to display the notification to false when the button is clicked.
